app/assets/stylesheets/admin/loans/timeline/_table.scss

```diff
@@ -1,11 +1,34 @@
 @import "../../colors";
 
 .timeline-table {
-  table {
+  .date-col {
+    min-width: 8rem;
+  }
+
+  // Specificity used to override Bootstrap defaults
+  .table thead:first-child tr {
+    th {
+      border: 1px solid $table-border-color;
+      border-bottom-width: 2px;
+    }
+  }
+
+  td, th {
     background: $white;
   }
 
-  .date-col {
-    min-width: 8rem;
+  td {
+    border: 1px solid $table-border-color;
+
+    &.spacer {
+      background: $background-gray;
+      border: none;
+    }
+  }
+
+  .fa-cog {
+    color: $gray3;
+    font-size: 1.2em;
```

Comment:
Can these be a little lighter and smaller please. They have too much visual weight currently. The viewer's focus should be on the name of the group. They just need to be able to find the settings icon if they are seeking it.